packages/squiggle-lang/src/rescript/ReducerInterface/ReducerInterface_GenericDistribution.res

```diff
@@ -162,6 +162,38 @@ module Helpers = {
       }
     }
   }
+  let constructNonNormalizedPointSet = (
```

Comment:
Can we remove this? It's not being used. Also, I don't really get what it's doing (and it would probably make sense in a different file, like ``GenericDist``.